### PR TITLE
dreamview: fix bug at sim control

### PR DESCRIPTION
**Pull request by qwetqwe**
InitStartPoint() can handle  using localization point and using dummy point.if localization is not empty, program will jump here.

### CODE REVIEW

**Comment by storypku** [comment]
@lj0919 Please help review this PR. Thank you.

**Comment by lj0919** [comment]
@qwetqwe https://github.com/ApolloAuto/apollo/blob/f02eda58ca1abf939c567fc8db3a461fcbcbc95a/modules/dreamview/backend/sim_control/sim_control.cc#L255 when localization reader has no data, set_start_point is passed in true, we will init start point

**Comment by qwetqwe** [comment]
Hi, @lj0919 
In SimControl::InitStartPoint ,the function seems that if localization not empty, sim control will use lastest localization message.
https://github.com/ApolloAuto/apollo/blob/f02eda58ca1abf939c567fc8db3a461fcbcbc95a/modules/dreamview/backend/sim_control/sim_control.cc#L140-L164
The function explaination :
https://github.com/ApolloAuto/apollo/blob/f02eda58ca1abf939c567fc8db3a461fcbcbc95a/modules/dreamview/backend/sim_control/sim_control.cc#L120-L121
But program can not get into this branch, and set start point  when localization not empty, and will output localization 0 0 0 as default.

**Comment by lj0919** [comment]
@qwetqwe  Hi, Sorry for the late reply, I misunderstood what you meant at first, the else branch looks really redundant.

File: modules/dreamview/backend/sim_control/sim_control.cc [hunk 1]
```diff
@@ -109,7 +109,7 @@ void SimControl::InitTimerAndIO() {
 
 void SimControl::Init(bool set_start_point, double start_velocity,
```
Comment (lj0919) [inline modules/dreamview/backend/sim_control/sim_control.cc:110]: The first argument set_start_point is no longer needed, remove it to make the function cleaner,  then test to make sure there is no problem with sim_control

**Review by lj0919 — CHANGES_REQUESTED** [review]
Remove unused argument to make the function cleaner

**Review by lj0919 — APPROVED** [review]

**Pull request merged**